istioctl/pkg/writer/envoy/configdump/cluster.go

```diff
@@ -169,10 +172,10 @@ func (c *ConfigWriter) retrieveSortedClusterSlice() ([]*cluster.Cluster, error)
 	return clusters, nil
 }
 
-func safelyParseSubsetKey(key string) (model.TrafficDirection, string, host.Name, int) {
-	if len(strings.Split(key, "|")) > 3 {
+func safelyParseSubsetKey(key string) (model.TrafficDirection, string, host.Name, int, model.TunnelType) {
+	if len(strings.Split(key, "|")) > 4 {
```

Comment:
We need to support the old format as well for backwards compatibility

Reply:
fixed